db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_11_10_172116) do
+ActiveRecord::Schema.define(version: 2021_11_02_182613) do
```

Comment:
I noticed the diff in this file includes schema changes not related to this PR. Can you remove those changes?